packages/node_modules/@webex/internal-plugin-device/src/features/feature-model.js

```diff
@@ -21,6 +21,8 @@ import {FEATURE_TYPES} from '../constants';
  */
 const FeatureModel = AmpState.extend({
 
+  idAttribute: 'key', // needed by Ampersand to determine unique item
```

Comment:
Yay for ampersand magic!
![img](https://media3.giphy.com/media/Sf0lxerEx2eNG/giphy.gif)